.travis.yml

```diff
@@ -1,8 +1,30 @@
 language: python
-python:
-  - "2.7"
-  - "3.4"
-  - 3.5
+dist: trusty
+sudo: required
```

Comment:
I don't think we need either sudo or the `dist` specification here, we can just let it use the defaults except for 3.7 and 3.8.